orchestrator/db/models.py

```diff
@@ -169,15 +169,16 @@ class ProcessSubscriptionTable(BaseModel):
 
 class ProductTable(BaseModel):
     __tablename__ = "products"
-
-    product_id = Column(UUIDType, server_default=text("uuid_generate_v4()"), primary_key=True)
-    name = Column(String(), nullable=False, unique=True)
-    description = Column(Text(), nullable=False)
-    product_type = Column(String(255), nullable=False)
-    tag = Column(String(TAG_LENGTH), nullable=False, index=True)
-    status = Column(String(STATUS_LENGTH), nullable=False)
-    created_at = Column(UtcTimestamp, nullable=False, server_default=text("current_timestamp()"))
-    end_date = Column(UtcTimestamp)
+    __allow_unmapped__ = True
```

Comment:
What happens without `__allow_unmapped__`?

From what I'm [reading in the guide](__allow_unmapped__) this should only be needed for models that don't use mapped columns, but all columns here are mapped (I think?)

Reply:
Is it because of the functions perhaps?

Reply:
I'll double check. I think it's only needed now in SubscriptionInstanceTable because of a 'free' variable.